src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
+  font-family: 'Roboto', sans-serif;
   margin: 0;
+  background-color: gainsboro;
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.moyo-header{
+  display: flex;
+  width: auto;
+  height: 60px;
+  background-color: white;
+  justify-content: space-between;
+}
+
+.image {
+  display: flex;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```

Comment:
You can add padding: 0px 50px; for header instead of using margin left/right for image and navigation